pkg/antctl/raw/check/installation/command.go

```diff
@@ -103,9 +106,16 @@ type testContext struct {
 }
 
 func Run(o *options) error {
+	var runFilterRegex *regexp.Regexp
+	if re, err := regexp.Compile(o.runFilter); err != nil {
```

Comment:
Should it check if `runFilter` is empty, not sure if an empty pattern can match everything but I assume the intent was leaving it as nil for empty input given the following `runFilterRegex != nil` check?

Reply:
Yes, you're right, that was the intent.
An empty pattern will indeed match everything (which is why the CI logs show that all the tests are running), but there is no reason to rely on that here. I will update the PR.